commands/bootstrap/inputs.go

```diff
@@ -170,11 +170,11 @@ var (
 	jdkDefaultInputs = map[string]defaultInputType{
 		"jdk.download_url": defaultInputType{
 			description: "Java Development Kit download URL",
-			value:       "https://download.oracle.com/otn-pub/java/jdk/8u202-b08/1961070e4c9b4e26a04e7f5a083f551e/jdk-8u202-linux-x64.tar.gz",
+			value:       "https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl=hotspot&os=linux&arch=x64&release=jdk8u212-b03&type=jdk",
```

Comment:
This point to https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jdk_x64_linux_hotspot_8u212b03.tar.gz
I think I prefer this url  (more readable, less error-prone), WDYT?

Reply:
Not sure if they decide to move their releases on Bintray or other download platform. Maybe their API is more stable ?

Reply:
Ok, my concern was that when used this value should be properly quoted in order to prevent interpreting `&` characters by a shell.